### ISSUE TITLE
A couple of fixes: add quote and revised argument name

### ISSUE THREAD

**Issue by ggurkan** [lmorabit/long_baseline_pipeline#98]
Hi,
delay_sols should be saved as string in facetselfcal_config.txt and facetselcal.py expects preapplyH5_list as an argument not preapplyH5-list so the following line could be added to run_selfcal.py to fix these issues:

    os.system("sed -i 's!"+delay_sols+"!"+'"'+delay_sols+'"'+"!g' facetselfcal_config.txt")
    os.system("sed -i 's!preapplyH5-list!preapplyH5_list!g' facetselfcal_config.txt")

after the following line:
os.system( 'sed -i "s~DELAYCAL_SOLUTIONS~{:s}~g" {:s}'.format(delay_sols, destfile))

**Comment by lmorabit** [comment]
Hi Gulay, at what point in the pipeline are you having this error? Is it in Split-Directions?

**Comment by ggurkan** [comment]
Yes, in the split-directions step and I sorted it out using the lines above.

**Comment by lmorabit** [comment]
Okay, thanks for pointing this out.  It needs a different solution than the one you suggest, as run_selfcal.py is used by both Delay-Calibration and Split-Directions and needs to work for both.  I'll work on a fix.
